apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/TradeStateController.java

```diff
@@ -137,31 +137,34 @@ public void onActivate() {
                     }));
 
             errorMessagePin = bisqEasyTrade.errorMessageObservable().addObserver(errorMessage -> {
```

Comment:
If error code is set on error message change we have dependency on order how those 2 fields are set. I will look into alternative impl. by modelling the error data into an observable object.